lib/orderbook/OrderBook.ts

```diff
@@ -354,15 +355,28 @@ class OrderBook extends EventEmitter {
       };
     }
 
-    if (!this.nosanitychecks) {
-      // check if sufficient outbound channel capacity exists
-      const { outboundCurrency, outboundAmount } = Swaps.calculateInboundOutboundAmounts(order.quantity, order.price, order.isBuy, order.pairId);
-      const swapClient = this.swaps.swapClientManager.get(outboundCurrency);
-      if (!swapClient) {
+    if (this.swaps && !this.nosanitychecks) {
```

Comment:
We don't need the `this.swaps` check any more since it's no longer nullable. Also don't we want to make the client checks even if `nosanitychecks` is true? We check it again further down anyway, so I think we can just drop this if statement.

Reply:
why remove `this.nosanitychecks` there are sanity checks.